data/mods/Russ_Extra_Stuff/construction.json

```diff
@@ -0,0 +1,30 @@
+[{
+        "type" : "construction",
+        "description" : "Start Vehicle Construction (alt)", "//": "For superalloy folding frames", 
+        "category" : "OTHER",
+        "difficulty" : 0,
+        "time" : 10,
+        "components" : [
+            [
+                [ "alloy_foldframe", 1 ]
+            ]
+        ],
+        "pre_special" : "check_empty",
+        "post_special" : "done_vehicle"
+    },
+{
+        "type" : "construction",
+        "description" : "Set up a honeybee hive", 
+        "category" : "OTHER",
+        "difficulty" : 0,
+        "time" : 20,
+        "components" : [
+            
+                [[ "bee_skep", 1 ]],
+	 	[[ "queen_bee",1 ]]
+            
+        ],
+        "pre_special" : "check_empty",
+	"post_terrain" :"t_honeybee_hive_empty"
```

Comment:
You've got tabs here. Replace them with 4 spaces. Applies to entire code (use search and replace or something like that).